form.html

```diff
@@ -0,0 +1,188 @@
+                          type="radio" 
+                          name="voute" 
+                          value="y"
+                        >
+                        Yes
+                    </label>
+                    <label>
+                        <input 
+                          type="radio" 
+                          name="voute" 
+                          value="n"
+                        >
+                        No
+                    </label>
+                </div>
+                <div class="form-field">
+                    <label>
+                        What is your favorite color?
+                        <input 
+                          type="color" 
+                          name="color" 
+                          required
+                        >
+                    </label>  
+                </div>
+                <div class="form-field">
+                    <label>
+                        What time do you go to bed?
+                    <input 
+                      type="text" 
+                      name="question" 
+                      placeholder="--:--:-- --" 
+                    >
+                    </label>   
+                </div>
+                <div class="form-field">
+                    <label>
+                        What are your favorite brand of cars?
+                        <select name="car" required multiple>
+                            <option value="BMW">BMW</option>
+                            <option value="Audi">Audi</option>
+                            <option value="Lada">Lada</option>
+                        </select>
+                    </label>
+                </div>
+                <div class="form-field">
+                    <label>
+                        How do you rate our work?
+                    <input 
+                      type="range" 
+                      name="rate" 
+                      required
+                    >
+                    </label>  
+                </div>
+            </fieldset>
+        </section>
+        <section class="container">
+            <fieldset>
+                <legend>Additional info</legend>
+                <div class="form-field">
+                    <label>
+                        Comments:
+                        <textarea
+                        name="comments"
+                        ></textarea>
+                    </label>
+                </div>
+                <div class="form-field">
+                    <label>
+                        Wold you recommend us?
+                        <select name="voute" required>
+                            <option value="Yes">Yes</option>
+                            <option value="No">No</option>
+                        </select>
+                    </label>   
+                </div>
+            </fieldset>
+        </section>
+        <button class="form-field" type="submit">Submit</button>
+    </form>
+
+</body>
+</html>
```

Comment:
```suggestion
</html>

```
Пустий рядок в кінці